scio-parquet/src/main/scala/com/spotify/scio/parquet/read/ParquetReadFn.scala

```diff
@@ -54,10 +55,35 @@ class ParquetReadFn[T, R](
   private val logger = LoggerFactory.getLogger(this.getClass)
 
   private lazy val granularity =
```

Comment:
since we now have 2 granularity settings (split and filter). I think we should disambiguate here